### PR TITLE
Newlin/enhance transactions

### PR DESCRIPTION
**Pull request by selfnewlin**
This adds new fields to the transactions and enhance transactions request/responses

### CODE REVIEW

File: openapi/mx_platform_api_beta.yml [hunk 1]
```diff
@@ -578,15 +591,27 @@ components:
           example: false
           nullable: true
           type: boolean
+        is_subscription:
+          example:
```
Comment (brettmortensen) [inline openapi/mx_platform_api_beta.yml:595]: I noticed the example value was missing here.

File: openapi/mx_platform_api_beta.yml [hunk 2]
```diff
@@ -578,15 +591,27 @@ components:
...
         merchant_category_code:
           example: 5411
           nullable: true
           type: integer
         merchant_guid:
-          example: MCH-7ed79542-884d-2b1b-dd74-501c5cc9d25b
+          example: MCH-14f25b63-ef47-a38e-b2b6-d02b280b6e4e
+          type: string
+        merchant_location_guid:
+          example:
```
Comment (brettmortensen) [inline openapi/mx_platform_api_beta.yml:610]: Missing an example value here.

File: openapi/mx_platform_api_beta.yml (hunk 1 above)
Comment (selfnewlin) [reply openapi/mx_platform_api_beta.yml]: fixed.

File: openapi/mx_platform_api_beta.yml (hunk 2 above)
Comment (selfnewlin) [reply openapi/mx_platform_api_beta.yml]: Thanks! Fixed.

**Comment by brettmortensen** [comment]
:shipit:

**Pull request merged**